src/components/skeletons/ProductCardSkeleton/product-card-skeleton.scss

```
@@ -0,0 +1,72 @@
+@import "../../../styles/vendors/include-media.scss";
+
+[data-store-product-card-skeleton] {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  min-width: 14vw;
+  height: 100%;
+  padding: var(--space-1) var(--space-1) var(--space-2);
+  overflow: hidden;
+  border: 1px solid transparent;
+  border-radius: var(--border-radius-default);
+
+  @include media(">=notebook") { min-width: 12rem; }
+
+  &[data-bordered="true"] { border: 1px solid var(--color-border-display); }
+
+  &[data-variant="vertical"] {
+    [data-card-content] { padding-top: var(--space-2); }
+  }
+
+  &[data-variant="horizontal"] {
+    [data-card-content] {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+    }
+  }
+
+  [data-product-card-skeleton-image] {
+    [data-element-variant="image"] {
+      height: 45vw;
+
+      @include media(">=notebook") {
+        height: 195px;
+      }
+    }
+
+    &[data-sectioned="true"] {
+      [data-element-variant="image"] {
+        min-width: 13rem;
+        height: 195px;
+
+        @include media(">=notebook") { height: 195px; }
+      }
+    }
+  }
+
+  [data-product-card-skeleton-content] {
+    position: relative;
+    height: 20%;
+    margin-top: var(--space-2);
+
+    [data-element-variant="text"] { margin-bottom: var(--space-2); }
+
+    [data-element-variant="button"] { margin-top: var(--space-4); }
+
+    [data-element-variant="badge"] {
+      min-width: 30%;
+
+      @include media(">=notebook") {
+        min-height: 30px;
```

```suggestion
        min-height: rem(30px);
```
or var(--space-5) = 32px